test/powershell/Modules/Microsoft.PowerShell.Core/CompatiblePSEditions.Module.Tests.ps1

```diff
@@ -538,6 +553,63 @@ Describe "Additional tests for Import-Module with WinCompat" -Tag "Feature" {
             $out | Should -BeExactly 'CouldNotAutoloadMatchingModule'
         }
     }
+
+    Context "Tests around PSModulePath in WinCompat process" {
+        BeforeAll {
+            $pwsh = "$PSHOME/pwsh"
+            Add-ModulePath $basePath
+            $ConfigPath = Join-Path $TestDrive 'powershell.config.json'
+        }
+
+        AfterAll {
+            Restore-ModulePath
+        }
+
+        AfterEach {
+            Get-Module $allModules | Remove-Module -Force
+        }
+
+        It 'WinCompat process does not inherit PowerShell-Core-specific paths' {
```

Comment:
Looks like other test cases in this file explicit skip non-Windows, should you do this here?

Reply:
For these new tests this is already done in their `Describe -> BeforeAll`.
However the last test that was added had copy-paste leftover in a form of the test-level `-Skip`, so I cleaned that.